app/models/miq_server/log_management.rb

```diff
@@ -56,13 +56,13 @@ def post_logs(options)
     raise _("Log depot settings not configured") unless context_log_depot
     context_log_depot.update(:support_case => options[:support_case].presence)
 
+    current_logfile = post_current_logs(taskid, context_log_depot)
```

Comment:
Doesn't this cause us to re-use the LogFile row?

Previously, I believe we got one per `post_*` method call and we'd have status information and other metadata about it. I think.